rotkehlchen/db/dbhandler.py

```diff
@@ -3457,3 +3461,25 @@ def create_db_backup(self) -> Path:
             new_db_path,
         )
         return new_db_path
+
+    def add_connected_location(self, location: Location) -> None:
```

Comment:
This is not needed and neither is calling this every time you add a new entry needed :sweat_smile: 

You already have this data.

So you add 1k trades with location COINBASE
And then with this you add COINBASE to connected locations 1k times too.

See the redundancy? Remove the entire thing.